assets/css/style.css

```diff
@@ -50,16 +44,23 @@ h1, h2, h3, h4, h5, h6 {
 }
 
 .nav{
-    font-size: 20px;
+    font-size: 100%;
     text-transform: uppercase;
 
+}
+
+
+.nav_text{
+    font-size: 100%;
+    text-transform: uppercase;
+    color: #fff;
     
 }
 
 .nav__link{
     display: inline-block;
     vertical-align: top;
-    margin: 0 20px;
+    margin: 0 1.2vw;

```

Comment:
still no, margin size  should not be depended on view port, value should be stable, px or rem.